examples/with-antd-mobile/pages/_document.js

```diff
@@ -1,11 +1,30 @@
 import Document, { Head, Main, NextScript } from 'next/document'
 
+// Note that fastclick is here because it is in the antd-mobile docs
+// but it may not be required: https://github.com/ant-design/ant-design-mobile/issues/576
+
 export default class extends Document {
   render () {
     return (
       <html>
         <Head>
-          <script src='/static/hd.min.js' />
+
+          <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no" />
+          <script src="https://as.alipayobjects.com/g/component/fastclick/1.0.6/fastclick.js"></script>
```

Comment:
I am not sure if we really should add fastclick. Afaik it has been deprecated.

https://developers.google.com/web/updates/2013/12/300ms-tap-delay-gone-away

Reply:
antd has it in their official directions so I felt like matching that was playing it safe: https://mobile.ant.design/docs/react/introduce#3.-Usage

The ticket linked to in the "usage" area seems to imply that it is not necessary but does resolve some problem: https://github.com/ant-design/ant-design-mobile/issues/576

I defer to whatever you ultimately think we should do because its over my head.